components/integrations_marketplace/marketplace_item/marketplace_item.jsx

```diff
@@ -140,12 +141,24 @@ export default class MarketplaceItem extends React.Component {
             const ariaLabel = `${this.props.name}, ${this.props.description}`.toLowerCase();
             const versionLabel = `(${this.props.version})`;
 
+            let pluginIcon;
+            if (this.props.iconUrl) {
+                pluginIcon = (
+                    <img
+                        src={this.props.iconUrl}
+                        style={{width: 32, height: 32}}
```

Comment:
To be consistent, could we use a css class here? Possibly we could use `className='icon__plugin'` and add `.svg` to the `.scss` [here](https://github.com/mattermost/mattermost-webapp/blob/c88394285d41e62c0f84cf2f520b27d0e10862e7/components/integrations_marketplace/marketplace_modal.scss#L114)